src/nrepl/bencode.ts

```diff
@@ -108,7 +108,9 @@ class BIncrementalDecoder {
     state: State = { id: 'ready' };
     stack: State[] = [];
 
-    constructor() {}
+    constructor() {
+        // do nothing
+    }
```

Comment:
this one can just go away, right?

Reply:
I think so? I confess I don't do classes much and I assumed this was empty for a good reason.